engine-tests/src/test/java/org/terasology/logic/console/ConsoleTest.java

```diff
@@ -24,6 +24,8 @@
 
 public class ConsoleTest extends TerasologyTestingEnvironment {
 
+    private final String MSG_TXT = "Test message";
```

Comment:
Let's not abbreviate please: 

`private final String MESSAGE_TEXT = "Test message";`

Reply:
I could be wrong, but it's fine to abbreviate constants, e.g. MAX_SPEED. But probably I'm not so close to local code styles. Let's fix it.

Reply:
The general idea is to make sure people can read the code without having to de-crypt the code in their head first. Of course plenty of acronyms and abbreviations are widely used, i.e. min/max. But if, as a habit, we avoid abbreviation, our code is closer to plain language and requires fewer brain cycles to deal with, leaving room for more important issues to think about.